armi/materials/material.py

```diff
@@ -845,9 +851,36 @@ class FuelMaterial(Material):
     class1_wt_frac = None
     class1_custom_isotopics = None
     class2_custom_isotopics = None
-    puFrac = 0.0
-    uFrac = 0.0
-    zrFrac = 0.0
+    puFracDefault = 0.0
+    uFracDefault = 0.0
+    zrFracDefault = 0.0
```

Comment:
Adding plutonium, uranium, and zirconium to a generic `FuelMaterial` seems odd. We should discuss.

Reply:
Probably outside of the scope of this change, just a comment. Similar comment for @john-science in the `duplicate` method.